examples/hitl/rearrange_v2/ui.py

```diff
@@ -696,20 +705,29 @@ def _update_selected_object_ui(self):
                 # Get the contextual information.
                 color_ui_valid = [0.2, 1.0, 0.2, 1.0]
                 color_ui_invalid = [1.0, 0.2, 0.2, 1.0]
+                color_ui_info = None
+                color_ui_object_info = [0.8, 0.8, 0.2, 1.0]
                 if self._is_object_pickable(object_id):
                     if self._held_object_id == None:
                         if self._world.is_any_agent_holding_object(object_id):
-                            contextual_info = (
-                                "The object is held by another agent."
+                            contextual_info.append(
+                                (
+                                    "The object is held by another agent.",
+                                    color_ui_invalid,
+                                )
                             )
-                            contextual_color = color_ui_invalid
                         else:
                             if self._is_within_reach(obj.translation):
-                                contextual_info = "Double-click to pick up."
-                                contextual_color = color_ui_valid
+                                contextual_info.append(
+                                    (
+                                        "Double-click to pick up.",
```

Comment:
should this be color_ui_info?

Reply:
The color theme is the following:
* White: Generic information.
* Yellow: Anything related to a pickable object.
* Cyan: Selection.
* Green: Anything related to a possible action.
* Red: Anything related to an impossible action.

I put this label green to match with the green highlight that indicates that an object can be picked up.

https://github.com/user-attachments/assets/ba312739-3802-4d27-b1d7-9c2c03671260